### PR TITLE
Add about us modal

### PR DESCRIPTION
**Pull request by itspatys**
Addresses: https://github.com/instructlab/ui/issues/102

https://github.com/user-attachments/assets/98d4abcb-6368-449a-8eda-e9fc175816fa

Please let me know what you think. I've tried making some of the text lighter but for some reason changing `font-weight` to `lighter` or `300` doesn't work.

### CODE REVIEW

**Comment by vishnoianil** [comment]
@itspatys This looks great. We should use the x.y.z formate for the version, so lets use 1.0.0 for version.

@Misjohns Please take a look and share your thoughts.

File: src/components/AboutModal/AboutModal.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            </Text>
+            <Text className={styles.description}>
+              InstructLab is an open source AI project that allows you shape the future of Large Language Models. Join the community to start
+              contributing today.
+            </Text>
+          </TextContent>
+        </div>
+        <div className={styles.joinCommunityButtonWrapper}>
+          <Button variant="primary" icon={<GithubIcon />} iconPosition="left" size="lg" className={styles.joinCommunityButton}>
+            Join the community
```
Comment (aevo98765) [inline src/components/AboutModal/AboutModal.tsx:39]: @vishnoianil @nerdalert  do we have a way to link people to join the instructlab community? At present this does not link the user to anywhere.

Comment (vishnoianil) [reply src/components/AboutModal/AboutModal.tsx:39]: @aevo98765 we should point it to https://github.com/instructlab

**Comment by Misjohns** [comment]
Looks good just a few minor tweaks. Some maybe fine as is but it's hard for me to accurately judge color on video:

- The close X looks big. Should be 24px
- About InstructLab should be 28pt medium #DEDEDD
- InstructLab is an open source... should be 18pt #FFFFFF and wrap like sample below
- Join the Community should be 16pt medium #FFFFFF and button should be black #030303
- Links and legal should be 14pt #FFFFFF

> InstructLab is an open source AI project that allows
you to shape the future of Large Language Models.
Join the community to start contributing today.

**Comment by aevo98765** [comment]
> Looks good just a few minor tweaks. Some maybe fine as is but it's hard for me to accurately judge color on video:
> 
> * The close X looks big. Should be 24px
> * About InstructLab should be 28pt medium #DEDEDD
> * InstructLab is an open source... should be 18pt #FFFFFF and wrap like sample below
> * Join the Community should be 16pt medium #FFFFFF and button should be black #030303
> * Links and legal should be 14pt #FFFFFF
> 
> > InstructLab is an open source AI project that allows
> > you to shape the future of Large Language Models.
> > Join the community to start contributing today.

Hey @Misjohns , is there a way we can access these specific requirements as devs so we can implement these specifics as a first pass? Could we detail these as requirements in the issue?

**Comment by Misjohns** [comment]
I can either give you access to the Figma file or create a quick red line to spec fonts, colors, etc. Which is better for you?

**Comment by aevo98765** [comment]
> I can either give you access to the Figma file or create a quick red line to spec fonts, colors, etc. Which is better for you?

Let's chat about it in the community meeting tomorrow. This would be good to have a consistent approach going forward as I think we should take as much guidance as possible from you guys early on.

**Comment by vishnoianil** [comment]
> > I can either give you access to the Figma file or create a quick red line to spec fonts, colors, etc. Which is better for you?
> 
> Let's chat about it in the community meeting tomorrow. This would be good to have a consistent approach going forward as I think we should take as much guidance as possible from you guys early on.

+1, @aevo98765 / @Misjohns Added this as an agenda item for community meeting.

**Comment by vishnoianil** [comment]
@itspatys If you can address @Misjohns review comments on this PR, i think it's good to go.

**Comment by vishnoianil** [comment]
> @vishnoianil I will try, but currently I am on vacation and I am not sure if I will have the time but I will try!

no worries at all, enjoy your vacations, last thing i want is folks to look at their laptop during their day off  :-)

**Comment by aevo98765** [comment]
> @vishnoianil I will try, but currently I am on vacation and I am not sure if I will have the time but I will try!

As @vishnoianil said. Prioritise your vacation! I have completed the changes and also caught (and sorted) some issue with the AboutModal not having an aria-label for accessibility. Going to push this one through.

**Comment by aevo98765** [comment]
@vishnoianil @Misjohns looks like I can't review now. When you get chance I would appreciate a review. Thanks!

**Comment by Misjohns** [comment]
Placed the changes beside design mockup so you can easily compare
image

**Comment by Misjohns** [comment]
Link to Figma dev mode for About screen: https://www.figma.com/design/6QLf45DArE6d3ngxRSYarO/InstructLab?node-id=73-7952&m=dev

Let me know if you can access it.

**Review by vishnoianil — APPROVED** [review]
LGTM

**Pull request merged**